perfscale_regression_ci/scripts/scalability/master-failover-two-master-stop-scale.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
+
+##############################################################################
+# Author: skordas@redhat.com
```

Comment:
@qiliRedHat do you want to add yourself as a second author here?

Reply:
@paigerube14 No, I just copied Simon's code from his fork and did some tiny update to adapt to the new release.

Reply:
@skordas Sorry, I thought the code was on your fork based on some previous test run results and I used that code in the 4.16 test run... I just realized you have another script already in the svt master branch, but in another folder [openshift_scalability/scripts/master-failover-one-master-stop.sh](https://github.com/openshift/svt/blob/master/openshift_scalability/scripts/master-failover-one-master-stop.sh). But this script looks even older to me because it uses [cluster-loader](https://github.com/openshift/svt/blob/master/openshift_scalability/scripts/master-failover-one-master-stop.sh#L166). 
Would you confirm which one is the latest?
1. https://github.com/skordas/svt/blob/HA_test_scripts/openshift_scalability/scripts/master-failover-two-master-stop-scale.sh
2. https://github.com/openshift/svt/blob/master/openshift_scalability/scripts/master-failover-one-master-stop.sh

If you confirmed my new script is based on the correct one, I can delete [openshift_scalability/scripts/master-failover-one-master-stop.sh](https://github.com/openshift/svt/blob/master/openshift_scalability/scripts/master-failover-one-master-stop.sh) in this PR. If not, I will delete my new script and let you to do the update.

Reply:
This one.... perfscale_regression_ci/scripts/scalability/master-failover-two-master-stop-scale.sh looks correct.

[2] is for little different test case (OCP-22421)

[1] is supporting  OCP-22399.

You have made some small tweaks here - I think this one should stay here and you should add yourself as a coauthor.